### PR TITLE
Fixes part 2 and updates all parts

### LINKED ISSUE #301: Post: Building An Ember App With Rails Part 2
After changing the title to "Welcome to Boston Ember," the test is still failing with the following error:

screen shot 2015-03-10 at 7 49 34 am

I am running ember-cli v.0.1.15, if that helps. I noticed two other people in the comments had the same issue.

### PR DESCRIPTION
**Pull request by michaeldupuisjr**
Updates for Rails 4.2 and ember-cli 0.2.0.

Closes #301

### CODE REVIEW

File: source/posts/2014-05-08-building-an-ember-app-with-rails-part-2.md [hunk 1]
```diff
@@ -35,28 +41,30 @@ application over this multi-part series. Create a new directory and file
...
 
 module('Integration - Landing Page', {
-  setup: function() {
+  beforeEach: function() {
     App = startApp();
   },
-  teardown: function() {
-    Ember.run(App, 'destroy');
+  afterEach: function() {
+    Ember.run(App, App.destroy);
```
Comment (bcardarella) [inline source/posts/2014-05-08-building-an-ember-app-with-rails-part-2.md]: @rwjblue is this correct, I thought it would be `Ember.run(App, 'destroy')`

Comment (michaeldupuisjr) [inline source/posts/2014-05-08-building-an-ember-app-with-rails-part-2.md]: Ember-cli docs use `Ember.run (App, App.destroy)`. Not sure which is preferred...

Comment (rwjblue) [inline source/posts/2014-05-08-building-an-ember-app-with-rails-part-2.md]: The param can be either a string (in which case it is invoked on the "context") or a function (in which case it is invoked with the context enforced).

`Ember.run(App, 'destroy')` is roughly analogous to (without the extra closure):

```javascript
Ember.run(function() {
  App.destroy();
});
```

`Ember.run(App, App.destroy)` is roughly analogous to (without the extra closure):

```javascript
Ember.run(function() {
  App.destroy.call(App);
});
```

tldr; both work, but I would use `Ember.run(App, 'destroy')` as it is both easier to read and should be somewhat faster (no `.call`).

Comment (michaeldupuisjr) [inline source/posts/2014-05-08-building-an-ember-app-with-rails-part-2.md]: Thanks for the explanation @rwjblue. I will update!



Michael Dupuis
Developer
http://dockyard.com

**Comment by michaeldupuisjr** [comment]
Updated to use `Ember.run(App, 'destroy')` in `afterEach`.

**Comment by michaeldupuisjr** [comment]
Should be good for a merge @bcardarella.
